jf_agent/jf_jira/jira_download.py

```diff
@@ -755,17 +756,21 @@ def _download_jira_issues_page(
                 time.sleep(30)
                 if batch_size == 0:
                     if re.match(r"A value with ID .* does not exist for the field 'id'", e.text):
-                        return [], 1
+                        yield ([], 1)
+                        break
                     elif not get_changelog:
                         logging_helper.log_standard_error(
                             logging.WARNING, msg_args=[search_params], error_code=3062,
                         )
-                        return [], 0
+                        yield ([], 0)
+                        break
                     else:
                         get_changelog = False
                         batch_size = 100
                         start_at += 1
 
+    return [], 0
```

Comment:
This is the end case where we never yielded anything right? And also just the final return after yielding everything else?